README.md

```
@@ -20,3 +20,4 @@ and then:
 docker run -p 3000:3000 reto-winter-2025-ithaka-front
 ```
 # Deploy automático con GitHub Actions
+# test de trigger
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)